modules/ebi-metagenomics/foldcomp/decompress/main.nf

```diff
@@ -0,0 +1,47 @@
+process FOLDCOMP_DECOMPRESS {
+    tag "$meta.id"
+    label 'process_low'
+
+    conda "${moduleDir}/environment.yml"
+    container "${ workflow.containerEngine == 'singularity' && !task.ext.singularity_pull_docker_container ?
+        'https://depot.galaxyproject.org/singularity/foldcomp:0.0.5--h43eeafb_2':
+        'biocontainers/foldcomp:0.0.5--h43eeafb_2' }"
+
+    input:
+    tuple val(meta), path(fcz)
+
+    output:
+    tuple val(meta), path("{${fcz}_pdb,*.pdb,*.cif}"), emit: pdb
```

Comment:
This glob experssion looks strange, wouldn't 
```suggestion
    tuple val(meta), path("{*.pdb,*.cif}"), emit: pdb
```
be enough?
Also, it will have files `.cif` in the pdb output channel